adam-core/src/main/scala/org/bdgenomics/adam/rdd/GenomicRDD.scala

```diff
@@ -121,6 +125,130 @@ trait GenomicRDD[T, U <: GenomicRDD[T, U]] {
     replaceRdd(tFn(rdd))
   }
 
+  private[rdd] def sorted: Boolean
+  private[rdd] def partitionMap: Option[Seq[Option[(ReferenceRegion, ReferenceRegion)]]]
+
+  /**
+   * This repartition method repartitions all data in this.rdd and distributes it as evenly as possible
```

Comment:
`This repartition method repartitions all data in this.rdd...` &rarr; `Repartitions all data in this.rdd...`